plugins/home/backstage-plugin-home-markdown/src/MarkdownCard/Content.tsx

```diff
@@ -37,6 +44,28 @@ export type MarkdownContentProps = {
  * @public
  */
 export const Content = (props: MarkdownContentProps) => {
+  const githubApi = useApi(githubAuthApiRef);
```

Comment:
If you're feeling motivated this could be a separate custom hook (e.g. useGithubLoginState) ? Ideally we'd put it somewhere where it could be shared easily but I don't think we have anywhere... but extracting it might be the first step.

Reply:
Yea, good idea, we have a LOT of github plugins so probably I'll move it out, but in a separate PR, I'd like to merge this to not force the login popup in roadie.